inventory/host_vars/zuul.bonnyci-internal.portbleu.com

```diff
@@ -0,0 +1,3 @@
+---
+letsencrypt_csr_cn: zuul.bonnyci.com
```

Comment:
Hm, good point. It looks like we don't need internal https (unless there is a connection missing from http://bonnyci.org/lore/admins/architecture.html#connection-diagrams)